dashboard/lib/services/curriculum_pdfs/resources.rb

```diff
@@ -155,6 +155,15 @@ def generate_lesson_resources_title_page(lesson, directory = "/tmp/")
           end
           raise "File #{path.inspect} does not exist after generation" unless File.exist?(path)
 
+          if DCDO.get('use-ghostscript-to-generate-pdfs', false)
```

Comment:
This is for the cover pages between the lessons which have had issues with missing fonts.